examples/cmd/snapshotting/Makefile

```diff
@@ -50,28 +50,27 @@ bin/host-local: bin
 	$(call install_go,github.com/containernetworking/plugins/plugins/ipam/host-local,v1.1.1)
 
 vmlinux:
-	curl --location -o vmlinux https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/${ARCH}/kernels/vmlinux.bin
+	curl -L -o vmlinux https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/${ARCH}/kernels/vmlinux.bin
 
 firecracker:
 	curl -L ${RELEASE_URL}/download/${VER}/firecracker-${VER}-${ARCH}.tgz | tar -xz
 	mv release-${VER}-${ARCH}/firecracker-${VER}-${ARCH} firecracker
 	rm -rf release-${VER}-${ARCH}
 
+# Use rootfs and ssh key pre-built from S3 following firecracker getting-started guide
+# https://github.com/firecracker-microvm/firecracker/blob/main/docs/getting-started.md
 root-drive-with-ssh.img root-drive-ssh-key:
-	- mkdir temp
-	- git clone https://github.com/firecracker-microvm/firecracker temp
-	temp/tools/devtool build_rootfs
-	cp temp/build/rootfs/bionic.rootfs.ext4 root-drive-with-ssh.img
-	cp temp/build/rootfs/ssh/id_rsa root-drive-ssh-key
-	rm -rf temp
-
+	curl -L -o root-drive-with-ssh.img https://s3.amazonaws.com/spec.ccfc.min/ci-artifacts/disks/${ARCH}/ubuntu-18.04.ext4
```

Comment:
`devtool` is using ubuntu 22.04, correct? We should keep the same upgraded version here imo.

Reply:
Current HEAD of firecracker uses 22.04. Older versions of firecracker use 18.04.
Current issue is firecracker didn't publish `ubuntu-18.04.ext4` in the same S3 bucket. 
I can follow up with them and address this in a follow-up PR?